components/SampleAddressInput/SampleAddressInput.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * @author Felix Hildebrandt <fhildeb>
+ */
+
+import { useContext } from 'react';
+import { NetworkContext } from '../../contexts/NetworksContext';
+
+enum AddressType {
+  UP = 'UP',
+  Asset = 'Asset',
+}
+
+interface Props {
+  inputId: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const SampleAddressInput: React.FC<Props> = ({ inputId, onChange }) => {
+  const { network } = useContext(NetworkContext);
+
+  const changeInputAddress = (type: AddressType) => {
+    let address;
+    if (type === AddressType.UP) {
+      if (network.name === 'MAINNET') {
+        // TODO: Update to Mainnet Profile
+        address = '0x9139def55c73c12bcda9c44f12326686e3948634';
+      } else if (network.name === 'TESTNET') {
+        address = '0x9139def55c73c12bcda9c44f12326686e3948634';
+      }
+    } else if (type === AddressType.Asset) {
+      if (network.name === 'MAINNET') {
+        // TODO: Update to Mainnet Asset
+        address = '0x6395b330F063F96579aA8F7b59f2584fb9b6c3a5';
+      } else if (network.name === 'TESTNET') {
+        address = '0x6395b330F063F96579aA8F7b59f2584fb9b6c3a5';
+      }
+    }
+
+    if (address) {
+      const inputElement = document.getElementById(inputId) as HTMLInputElement;
```

Comment:
we are working with react here - there should be no need to use plain js like that:

`document.getElementById(inputId) as HTMLInputElement;`

Consider using react with: refs:

https://react.dev/learn/manipulating-the-dom-with-refs